stellar-contract-env-host/src/host.rs

```diff
@@ -806,6 +808,27 @@ impl Host {
             }
         }
     }
+
+    fn to_u256(&self, a: Object) -> Result<Uint256, HostError> {
+        self.visit_obj(a, |bin: &Vec<u8>| {
+            Ok(Uint256(
+                bin.as_slice()
+                    .try_into()
+                    .map_err(|_| HostError::General("bad u256 length"))?,
+            ))
+        })
+    }
```

Comment:
This function  shouldn't exist in the host. Either use the [defined object type](https://github.com/stellar/rs-stellar-contract-env/blob/main/stellar-contract-env-host/src/host_object.rs#L25-L26), or another alternative. 
Opened https://github.com/stellar/rs-stellar-contract-env/issues/157 to track this.